index.html

```diff
@@ -0,0 +1,126 @@
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">Store</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">Mac</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">iPod</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">iPhone</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">iPad</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">iTunes</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">Support</a>
+          </li>
+          <li class="grid right-nav-item">
+            <i class="fa fa-search fa grey-grad"><input type="text" class="topics grey-grad m5"></i>
+          </li>
+        </ul>
+      </nav>
+      <section class="main-text">
+        <a href="#">What will your verse be?</a>
+        <p>
+      Everyone has something to share. See how people are using
+       iPad in their unique ways to add to the world's story.
+        </p>
+      </section>
+    </section>
+    <aside class="promos">
+      <section class="grid2 promos-nav">
+          <section>
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c_2x.jpg" alt="iPhone 5c">
+          </section>
+          <section>
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s_2x.jpg"  alt="iPhone 5s. Forward thinking.">
+          </section>
+          <section>
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years_2x.jpg"  alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
+          </section>
+          <section>
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility_2x.jpg"  alt="Supplier Responsibility The latest results of our worldwide efforts.">
+          </section>
+      </section>
+    </aside>
+
+  <footer>
+    <section class="left">
+      <p>
+        Shop the <a href="#">Apple Online Store</a> (1-800-MY-APPLE), visit an <a href="#">Apple Retail Store</a>, or find a <a href="#">reseller</a>.
+      </p>
+      <p>
+        Copyright © 2014 Apple Inc. All rights reserved. <a href="#">Terms of Use</a> | <a href="#">Privacy Policy</a>
+      </p>
+    </section>
+    <section class="right">
+      <ul class="bottom-links">
+        <li>
+          <a href="#">Site Map</a>
+        </li>
+        <li>
+          |
+        </li>
```

Comment:
[**OPTIONAL**]
This is not the best way to implement this design, inspect the webpage on [apple.com](https://web.archive.org/web/20140301004610/http://www.apple.com/), for reference.